addons/patches/unscfoundries/CfgVehicles.inc

```diff
@@ -62,4 +62,78 @@ class CfgVehicles
       class Right : Right {};
     };
   };
+  class Splits_Pelican_base;
+  class Splits_UNSC_D77_TC_Pelican: Splits_Pelican_base
```

Comment:
Just for the sake of remembering this later because the patches can get cluttered easy;
```suggestion
  // fixing pylon mirroring and targetability; see https://github.com/WitchFreya/SWSAddons/pull/14
  class Splits_UNSC_D77_TC_Pelican: Splits_Pelican_base
```